src/Marten/Storage/DocumentTable.cs

```diff
@@ -178,7 +178,7 @@ public class DuplicatedFieldColumn: TableColumn
     {
         private readonly DuplicatedField _field;
 
-        public DuplicatedFieldColumn(DuplicatedField field) : base(field.ColumnName, field.PgType)
+        public DuplicatedFieldColumn(DuplicatedField field) : base(field.ColumnName, field.PgType, field.NotNull ? "NOT NULL": "NULL")
```

Comment:
**SUGGESTION:** Maybe it would be worth to make the `NOT NULL` and  `NULL` constants? I personally prefer to have them for such case, it's usually less error prone.